.vib/schema-registry/cypress/cypress/integration/schema_registry_spec.js

```diff
@@ -0,0 +1,35 @@
+/// <reference types="cypress" />
+import body from '../fixtures/schema.json';
+
+it('can access the API and obtain global compatibility level', () => {
+  cy.request({
+    method: 'GET',
+    url: '/config',
+    form: true,
+  }).then((response) => {
+    expect(response.status).to.eq(200);
+    expect(response.body.compatibilityLevel).to.contain('BACKWARD');
+  });
+});
+
+it('can access the API and create a new schema in the "test" subject', () => {
+  cy.request({
+    method: 'POST',
+    headers: { 'Content-Type': 'application/vnd.schemaregistry.v1+json' },
+    url: 'subjects/test/versions',
+    body: body,
+  }).then((response) => {
+    expect(response.status).to.eq(200);
+    expect(response.body.id).to.eq(1);
```

Comment:
Is this test retryable? That is, does it succeed two times in a row without issue? I'm concerned about both the API accepting two identical schemas and it returning the same `body.id` for 2 different executions.

Reply:
I'm not sure what you mean `is it retryable`.

The test should always return ID 1 unless the schema is changed or the POST request failed, it does not increment with every attempt if that is your concern.

I did test several consecutive executions and, as far as the schema does not change, the call will always return `id: 1`

See:
```
I have no name!@test-schema-registry-0:/$ curl -X POST -H "Content-Type: application/vnd.schemaregistry.v1+json" "http://localhost:8081/subjects/test/versions" -d '{"schema": "{\"type\": \"record\", \"name\": \"User\", \"fields\": [{\"name\": \"name\", \"type\": \"string\"}]}"}'
{"id":1}
I have no name!@test-schema-registry-0:/$ curl -X POST -H "Content-Type: application/vnd.schemaregistry.v1+json" "http://localhost:8081/subjects/test/versions" -d '{"schema": "{\"type\": \"record\", \"name\": \"User\", \"fields\": [{\"name\": \"name\", \"type\": \"string\"}]}"}'
{"id":1}
I have no name!@test-schema-registry-0:/$ curl -X POST -H "Content-Type: application/vnd.schemaregistry.v1+json" "http://localhost:8081/subjects/test/versions" -d '{"schema": "{\"type\": \"record\", \"name\": \"User\", \"fields\": [{\"name\": \"name\", \"type\": \"string\"}]}"}'
{"id":1}
#Changed the schema adding 'refs'
I have no name!@test-schema-registry-0:/$ curl -X POST -H "Content-Type: application/vnd.schemaregistry.v1+json" "http://localhost:8081/subjects/test/versions" -d '{"schema": "{\"type\": \"record\", \"name\": \"User\", \"fields\": [{\"name\": \"name\", \"type\": \"string\", \"refs\": \"this is an example\"}]}"}'
{"id":2}
```